examples/all-clusters-app/all-clusters-common/src/occupancy-sensing-stub.cpp

```diff
@@ -29,7 +29,7 @@ using namespace chip::DeviceLayer;
 using chip::Protocols::InteractionModel::Status;
 
 static std::unique_ptr<OccupancySensingAttrAccess>
-    gAttrAccess[MATTER_DM_OCCUPANCY_SENSING_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT];
+    gAttrAccess[MATTER_DM_OCCUPANCY_SENSING_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT + 1];

```

Comment:
Why is this `+1` needed? For fixing the crash on boot for all-clusters-app/esp32?

Reply:
Ughh!! This should not go in the PR. I just did this for testing, I will remove it.